core/types/block.go

```diff
@@ -93,7 +93,9 @@ type Header struct {
 	AuRaStep uint64
 	AuRaSeal []byte
 
-	BaseFee *big.Int `json:"baseFeePerGas"` // EIP-1559
+	BaseFee         *big.Int     `json:"baseFeePerGas"`   // EIP-1559
+	WithdrawalsHash *common.Hash `json:"withdrawalsRoot"` // EIP-4895
+	ExcessDataGas   *big.Int     `json:"excessDataGas"`   // EIP-4844
```

Comment:
Should we keep name & json name the same?

Reply:
ordinarily yes but for some reason they decided against this in the ongoing PR against geth:

https://github.com/ethereum/go-ethereum/blob/cc3d8dab3a8b02b7fd01384aa9f6ddd6b014e40f/cmd/evm/internal/t8ntool/block.go#L57

Reply:
🤣